index.html

```diff
@@ -54,6 +54,8 @@
             </li>
           </ul>
 
+          <h3><a href="companies">Ruby Companies in New Zealand</a></h3>
```

Comment:
The link looks a bit weird in the `h3` text.  I think it might look better as:

    <h3>Ruby Companies in New Zealand</h3>

    <p>Looking for help with a Ruby website in New Zealand?  Here is a link with some <a href="companies.html">pithy text</a></p>